src/components/ReactSlider/ReactSlider.jsx

```diff
@@ -177,6 +177,11 @@ class ReactSlider extends React.Component {
          */
         invert: PropTypes.bool,
 
+        /**
+         * Shows passed marks on the track
+         */
+        marks: PropTypes.arrayOf(PropTypes.number),
```

Comment:
It might be nice to provide some more options here -- for convenience so you don't have to build the array every time.

`bool`: when `true`, it will place a mark at every position there is a `step`
`number`: like `step`, it will place a mark at a given interval (e.g. have it step by 1, but only show a mark every 5)

What do you think? We can definitely release with just the support for array, but I think these other types would be pretty nice to have.

Reply:
You're right it is more practical with also just a boolean